packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -136,12 +147,27 @@ const ActivityDetailsSidebar = (props: Props) => {
   } = useDialogState()
 
   const [selectedUsers, setSelectedUsers] = React.useState<Option[]>([])
+  const selectedUser = selectedUsers[0]
+  const {organizations: viewerOrganizations} = viewer
+
+  const viewerOrganizationIds = new Set(viewerOrganizations.map((org) => org.id))
+
+  const selectedUserOrganizationIds = selectedUser?.organizationIds ?? []
+  const mutualOrgs = selectedUserOrganizationIds.filter((orgId) => viewerOrganizationIds.has(orgId))
+
+  const showOrgPicker = selectedUser && (mutualOrgs.length > 1 || !mutualOrgs.length)
+
+  const manuallySelectedOrgId = showOrgPicker && selectedOrgId !== '' && selectedOrgId
+  const firstMutualOrgId = mutualOrgs[0]
+  const defaultOrgId = selectedTeam.orgId
+  const newOneOnOneTeamOrgId = manuallySelectedOrgId
+    ? manuallySelectedOrgId
+    : firstMutualOrgId ?? defaultOrgId
 
   const oneOnOneTeamInput = selectedUsers[0]
     ? {
         email: selectedUsers[0].email,
-        // TODO: always pick default orgId for now, this will be changed in next PRs
-        orgId: selectedTeam.orgId
+        orgId: newOneOnOneTeamOrgId
```

Comment:
This PR introduces fully working version of one on one meeting.

Earlier I added `isOneOnOneTeam` flag here: https://github.com/ParabolInc/parabol/pull/8629/commits/f8a2d00188d7a91f183e38eb3e89f67c35bb289a

@tghanken Do we need to track any additional events?

Reply:
Can we add the `isOneOnOneTeam` property to the Segment events `New Team`, `Meeting Started`, `Meeting Joined`, and `Meeting Completed`?  I think that should cover it.

Reply:
@tghanken added ✅